src/Application/Console/Command/RunCommand.php

```diff
@@ -14,15 +14,17 @@
 
 use League\Event\Emitter;
 use League\Tactician\CommandBus;
+use phpDocumentor\Application\Configuration\ConfigurationFactory;
+use phpDocumentor\DomainModel\Dsn;
 use phpDocumentor\DomainModel\Parser\DocumentationRepository;
+use phpDocumentor\Infrastructure\FileSystemFactory;
```

Comment:
FileSystemFactory should be an interface in the application layer. Application cannot use the infrastructure layer directly